src/main/java/seedu/duke/exceptions/NegativeWithdrawalAmountException.java

```diff
@@ -0,0 +1,4 @@
+package seedu.duke.exceptions;
+
+public class NegativeWithdrawalAmountException extends Exception {
+}
```

Comment:
Same comment as in Message class that Negative withdrawal is a too specific of an exception